dali/test/example_test.cc

```diff
@@ -20,38 +20,57 @@ namespace testing {
 
 
 class ExampleOperatorTestCase : public DaliOperatorTest {
+ protected:
+  ExampleOperatorTestCase() :
+          tlcpu(std::unique_ptr<TensorList<CPUBackend>>(new TensorList<CPUBackend>())),
```

Comment:
```suggestion
          tlcpu(new TensorList<CPUBackend>()),
```
already handles explicit construction.